src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -75,11 +75,11 @@ internal override void ProcessResponse(HttpResponseMessage response)
         {
             ArgumentNullException.ThrowIfNull(response);
 
-            var baseResponseStream = StreamHelper.GetResponseStream(response);
+            Stream baseResponseStream = StreamHelper.GetResponseStream(response);
 
             if (ShouldWriteToPipeline)
             {
-                using var responseStream = new BufferingStreamReader(baseResponseStream);
+                using BufferingStreamReader responseStream = new(baseResponseStream);
```

Comment:
same here, we don't recommend this style change. The original code is `var xxx = new Ctor`, which makes it clear what type this variable is.

However, for new code, you can choose to use either style.

Reply:
I will revert, from other cleanup PRs I thought it was a desirable change

Reply:
For new code, you can use either style -- both are readable.
But for existing code, let's not change to avoid code churn.

Reply:
done